### DISCUSSION TITLE
PicoSpace's 5-Colour Method for Fast & Accurate Recolouring

### DISCUSSION THREAD

**Discussion by EmberSkyMedia** [likeproblem/RestockRecolour#18] [Show and tell]
Not sure this is helpful or not, but here is my method which seems to work pretty well.

I do an initial rough colour pallet in Blender so I can identfy all the parts. 
1. Load all the models that use that mesh, move them around so they don't clip each other, remove colliders
2. Load the PNG as the material so I can paint it.
3. Rough Paint the parts, I'm not using this but for now its for ID.

Colours:
Purple - Used to identify Parts, this tends to get painted over very quickly
Yellow - Mean DO NOT PAINT (basically structural parts or whatnot)
Red/Green/Blue - Parts to be Painted those Colours

Once everything is ID I move over to Affinity Photo 2 (or you could use Paint.net... whatever)

Load up the mesh as PNG, load up black background, load up all the SVG files you extracted earilier to ID where each UV zone is.
You want a blank top layer, this is what you will paint.

Paint the PNG with RGB based on your Blender PNG then convert the background (unpainted parts) Black. Do not copy over any remaining Purple or Yellow.

The 5 colour method is a lot faster than just using RGB I find and it allows me to quickly determine what colours I should use, Not sure this will help anyone but it sped up my process.

**Comment by likeproblem** [comment]
I am not exactly sure about Purple and Yellow color, but loading SVGs is actually very useful!

**Reply by EmberSkyMedia** [reply]
Yeah, it also helps to show if you missed something as some things in Blender are invisible until painted (I don’t know why… blender is the strangest program I have ever used).

**Reply by likeproblem** [reply]
I would guess it has to do with either depthmask or it rounds near transparent to transparent fully
